projects/laji/src/app/+project-form/project-form.component.ts

```diff
@@ -136,21 +135,21 @@ export class ProjectFormComponent implements OnInit, OnDestroy {
       startWith(this.router.url)
     );
 
-    this.showNav$ = combineLatest(this.browserService.lgScreen$, routerEvents$, this.isNavabarToggled$).pipe(
-      mergeMap(([isDesktopScreen, url, isNavbarToggled]) => 
+    this.showNav$ = combineLatest(this.browserService.lgScreen$, routerEvents$, this.isNavbarToggledSubject.asObservable()).pipe(
+      mergeMap(([isDesktopScreen, url, isNavbarToggled]) =>
       form$.pipe(
```

Comment:
@olzraiti i don't know if like that is good for you

Reply:
Yes that's good :) One more thing: could you remove the unused `isNavbarToggled$`? Then this would be ready to merge after Mikko accepts the pivotal task. Thanks for the effort!